pyracing/response_objects/upcoming_events.py

```diff
@@ -68,41 +68,36 @@ def __init__(self, dict):
         self.weather_wind_speed_units = dict['49']
 
 
-# I'm giving it my best guess here. This one has lots of nesting.
 class RaceGuide:
     def __init__(self, dict):
         self.catID = dict['catID']
         self.image = dict['image']
-        self.seasonSchedules = dict['seasonSchedules']
```

Comment:
Moved 5 lines down for consistency with other attributes that create subclass objects